15-components-lab_v16/src/01-timeconverter.jsx

```diff
@@ -1,12 +1,40 @@
-/**
- * Lab 01: 
- *
- * Implement a time converter component.
- * Component should show 3 input fields labeled hours, minutes and seconds.
- * Typing a value in either field updates the other two with corresponding values.
- */
-
-import React from 'react';
-
-export default React.createClass({
-});
+import React from 'react';
+

```

Comment:
1. Using `id` in React is considered bad practice. You never know how and when your component will be used. And for example if your component is used multiple times on the page, you get duplicate ids.

2. You saved 3 fields in `state`, but there's actually only one. The three numbers represent the same value in a different form (hours is just seconds divided by 3,600 for example). Please modify the code so you'll store only one item in `this.state`